docs/docs/setup-vast/install/linux.md

```diff
@@ -0,0 +1,116 @@
+# Linux
+
+Use our [pre-built packages](#download) below or [build VAST from
+source](/docs/setup-vast/build-from-source) to install VAST on any Linux
+distribution. To deploy VAST as system service, you can use our [systemd
+configuration](#systemd).
+
+To package VAST...
+
+## Download
+
+We offer pre-built packages containing a statically linked VAST binary, for the
+[latest release](https://github.com/tenzir/vast/releases/latest) and the current
+development version:
+
+<div align="center" class="padding-bottom--md">
+  <a class="button button--md button--primary margin--sm" href="https://github.com/tenzir/vast/releases/latest/download/vast-linux-static.tar.gz">Release</a>
+  <a class="button button--md button--info margin--md" href="https://github.com/tenzir/vast/releases/latest/download/vast-linux-static.tar.gz">Development</a>
+</div>
+
+You can also get static builds for a specific git commit. This involves
+navigating a bit through the github CI:
+
+1. Browse to the [VAST static workflow][vast-workflow]
+2. Click on the latest run, e.g., `Merge pull request...`
+3. Scroll to the end of the page
+4. Click on the artifact filename, e.g.,
+   `vast-v1.0.0-101-g6e7a4ef1a4-linux-static.tar.gz`
+
+[vast-workflow]: https://github.com/tenzir/vast/actions?query=branch%3Amaster+workflow%3A%22VAST%22
+
+## systemd
+
+VAST has native [systemd
+support](https://github.com/tenzir/vast/tree/master/systemd/) for running as
+system service. The service is sandboxed and runs with limited privileges.
+
+### Prepare the host system
+
+Please note that all subsequent commands require `root` privileges. The service
+requires a user and group called `vast`. You can create them as follows.
+
+```bash
+useradd --system --user-group vast
+```
+
+Make sure that you don't grant any special rights to this user, e.g., do not
+enable `sudo` or other privileged commands. Once the user exists, you should
+create the directory for VAST's persistent data and change the permissions such
+that it is owned by the new `vast` user:
+
+```bash
+mkdir -p /var/lib/vast
+chown -R vast:vast /var/lib/vast
+```
+
+The systemd unit passes a
+[vast.yaml](https://github.com/tenzir/vast/tree/master/systemd/) configuration
+file to the VAST process. Make sure that the new user can read the `vast.yaml`.
+
+### Configure the unit
+
+Before you begin, find the line beginning with `ExecStart=` at the very bottom
+of the `[Service]` section in the unit file. Depending on your installation path
+you might need to change the location of the `vast` binary and configuration
+file.
+
+```config
+ExecStart=/path/to/vast --config=/path/to/vast.yaml start
+```
+
+In case you plan to run a PCAP source directly inside the VAST server process
+via `vast spawn source pcap`, you need to make sure that the VAST process gets
+the required privileges to listen on the network interfaces.
```

Comment:
This feels a bit too specific, and since the CLI is changing in the future, we probably should rewrite this.

Reply:
Another way to set this up is to set `vast.start.commmands: [import pcap]`, which is a lot more intuitive and probably what we should be recommending instead.

Thinking of it further, it'd actually be nice to have some `vast remote-exec` command for simply executing a command in the context of a remote VAST instance. That could supersede `spawn source` a lot more generically, and would probably only take a few hours to gobble up. :thinking:

Reply:
> Another way to set this up is to set `vast.start.commmands: [import pcap]`, which is a lot more intuitive and probably what we should be recommending instead.

Yes, that's also possible. My point was rather to not mention start commands at all, because my hunch is they will be much less relevant after the Event Router.

> Thinking of it further, it'd actually be nice to have some `vast remote-exec` command for simply executing a command in the context of a remote VAST instance.

That's probably the no-longer-existent `vast send` command, which basically allowed for sending messages to component actors from via the CLI.

Reply:
It's actually slightly different from the `vast send` command in that it does not send a message to an actor, but rather only communicates via commands with the node, which is much safer. I reckon this would be a nice hackathon project, maybe for our next one. :)

Anyways, we can also erase the section. I'd rather not document it than mention `vast spawn`, because that command has absolutely no test coverage.